.bazelrc

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
This and other files in this PR are missing StableHLO license headers. Example: https://github.com/tensorflow/mlir-hlo/blob/master/stablehlo/CMakeLists.txt#L2.

Reply:
Done.